packages/ui/components/AppCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Info, ServiceInfoBadge } from './ServiceInfoBadge';
+
+interface AppCardProps {
+  isActive?: boolean;
+  name: string;
+  description: string;
+  badges: Info[];
+  isClient: boolean;
+  isServer: boolean;
+  className?: string;
+}
+
+export const AppCard = ({isActive = false, name, description, badges, className, isClient, isServer}:AppCardProps) => {
+  return (
+    <>
+      <div
```

Comment:
```suggestion
      <div
        aria-label={`${name} Application Card, ${isActive ? 'active' : 'inactive'}`}
```
@KhudaDad414 Could we also add aria-label for accessibility.